kafkaesque-core/src/main/java/in/rcard/kafkaesque/consumer/KafkaesqueConsumer.java

```diff
@@ -57,17 +58,20 @@
   }
 
   private KafkaConsumer<Key, Value> createKafkaConsumer(String brokersUrl) {
-    final Properties props = new Properties();
-    props.put(ConsumerConfig.GROUP_ID_CONFIG, "kafkaesque-consumer");
-    props.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
-    props.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, brokersUrl);
-    props.put(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, "true");
-    props.put(
-        ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, creationInfo.getKeyDeserializer().getClass());
-    props.put(
-        ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG,
-        creationInfo.getValueDeserializer().getClass());
-    final KafkaConsumer<Key, Value> consumer = new KafkaConsumer<>(props);
+    ConfigFactory.setProperty("configFile", "kafkaesque");
```

Comment:
Can't we set this property of the `ConfigFactory` away from here, i.e., in a place where it's set only once?